lms/djangoapps/api_manager/sessions/views.py

```diff
@@ -105,8 +111,16 @@ def post(self, request):
                     # violate our RESTfulness
                     #
                     engine = import_module(settings.SESSION_ENGINE)
-                    new_session = engine.SessionStore()
-                    new_session.create()
+                    if session_id is None:
+                        new_session = engine.SessionStore()
+                        new_session.create()
+                        success_status = status.HTTP_201_CREATED
+                    else:
+                        new_session = engine.SessionStore(session_id)
```

Comment:
@bradenmacdonald nit: it's not a new session - consider renaming the variable to just `session`